src/methods/status.rs

```diff
@@ -1,3 +1,28 @@
+//! Queries the status of a given node
+//!
+//! Returns the general status of a given node (sync status, nearcore node version, protocol version, etc), and the current set of validators.
+//!
+//! ## Example
+//! ```
+//! use near_jsonrpc_client::{methods, JsonRpcClient};
+//!
+//! # #[tokio::main]
+//! # async fn main() -> Result<(), Box<dyn std::error::Error>> {
+//! let client = JsonRpcClient::connect("https://rpc.testnet.near.org");
+//!
+//! let request = methods::status::RpcStatusRequest;
+//!
+//! let response = client.call(request).await;
+//!
+//! assert!(
+//!     matches!(response, Ok(methods::status::RpcStatusResponse { .. })),
+//!     "expected an Ok(RpcStatusResponse) but got [{:?}]",
+//!     response
+//! );
```

Comment:
We can drop the assertion message here.

```suggestion
//! assert!(matches!(
//!     status,
//!     Ok(methods::status::RpcStatusResponse { .. })
//! ));
```